py_june/settings.py

```diff
@@ -139,6 +141,22 @@
 
 LOCALE_PATHS = [os.path.join(BASE_DIR, "locale")]
 
+CKEDITOR_UPLOAD_PATH = "uploads/"
+CKEDITOR_CONFIGS = {
+    "default": {
+        # 'codeSnippet_theme': 'monkai',
+        "toolbar": "Basic",
+        # 'extraPlugins': ', '.join(
+        #     [
+        #         'codesnipets',
+        #         'widgets',
+        #         'dialogs',
+        #     ]
+        # ),
```

Comment:
add # TODO https://lambda-py.atlassian.net/browse/PJ-76 for commented configs
```suggestion
       "toolbar": "Basic",
       # TODO https://lambda-py.atlassian.net/browse/PJ-76
        # 'codeSnippet_theme': 'monkai',
        # 'extraPlugins': ', '.join(
        #     [
        #         'codesnipets',
        #         'widgets',
        #         'dialogs',
        #     ]
        # ),
```